browser/src/slideshow/SlideCompositor.ts

```diff
@@ -55,13 +49,7 @@ abstract class SlideCompositor {
 		return this._presentationInfo ? this._presentationInfo.slides.length : 0;
 	}
 
-	protected _getSlideWidth() {
-		return this._width;
-	}
-
-	protected _getSlideHeight() {
-		return this._height;
-	}
+	public abstract getCanvasSize(): Array<number>;
```

Comment:
Suggestion: Instead of Array<number>, can we use [number, number] with a comment indicating which element is the width and which is the height? This will show a more strict type.

Reply:
Let's change that another time...